2015-04-02 16:18:31,318 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:18:31,318 INFO - MQServer: Listener is starting...
 2015-04-02 16:18:31,326 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:18:31,418 INFO - Router: Verifying the message...
 2015-04-02 16:18:31,419 ERROR - ID:Alis-MacBook-Pro.local-62029-1427966310794-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59332-1427878014551-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQSessionExecutor.dispatch(ActiveMQSessionExecutor.java:134)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:205)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:18:32,467 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:18:32,467 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:18:32,468 INFO - Router: Verifying the message...
 2015-04-02 16:18:32,468 ERROR - ID:Alis-MacBook-Pro.local-62029-1427966310794-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59332-1427878014551-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:18:33,473 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:18:33,473 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:18:33,473 INFO - Router: Verifying the message...
 2015-04-02 16:18:33,474 ERROR - ID:Alis-MacBook-Pro.local-62029-1427966310794-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59332-1427878014551-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:18:34,479 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:18:34,479 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:18:34,479 INFO - Router: Verifying the message...
 2015-04-02 16:18:34,479 ERROR - ID:Alis-MacBook-Pro.local-62029-1427966310794-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59332-1427878014551-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:18:35,485 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:18:35,485 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:18:35,485 INFO - Router: Verifying the message...
 2015-04-02 16:18:35,485 ERROR - ID:Alis-MacBook-Pro.local-62029-1427966310794-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59332-1427878014551-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:18:36,497 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:18:36,497 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:18:36,498 INFO - Router: Verifying the message...
 2015-04-02 16:18:36,498 ERROR - ID:Alis-MacBook-Pro.local-62029-1427966310794-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59332-1427878014551-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:18:37,500 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:18:37,500 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:18:37,501 INFO - Router: Verifying the message...
 2015-04-02 16:18:37,501 ERROR - ID:Alis-MacBook-Pro.local-62029-1427966310794-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59332-1427878014551-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:18:37,502 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:18:37,502 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:18:37,503 INFO - Router: Verifying the message...
 2015-04-02 16:18:37,503 ERROR - ID:Alis-MacBook-Pro.local-62029-1427966310794-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59307-1427877990935-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQSessionExecutor.dispatch(ActiveMQSessionExecutor.java:134)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:205)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:18:38,504 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:18:38,504 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:18:38,506 INFO - Router: Verifying the message...
 2015-04-02 16:18:38,506 ERROR - ID:Alis-MacBook-Pro.local-62029-1427966310794-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59307-1427877990935-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:18:39,511 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:18:39,511 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:18:39,511 INFO - Router: Verifying the message...
 2015-04-02 16:18:39,512 ERROR - ID:Alis-MacBook-Pro.local-62029-1427966310794-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59307-1427877990935-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:18:40,518 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:18:40,519 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:18:40,519 INFO - Router: Verifying the message...
 2015-04-02 16:18:40,520 ERROR - ID:Alis-MacBook-Pro.local-62029-1427966310794-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59307-1427877990935-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:18:41,524 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:18:41,524 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:18:41,525 INFO - Router: Verifying the message...
 2015-04-02 16:18:41,525 ERROR - ID:Alis-MacBook-Pro.local-62029-1427966310794-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59307-1427877990935-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:18:42,527 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:18:42,527 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:18:42,528 INFO - Router: Verifying the message...
 2015-04-02 16:18:42,529 ERROR - ID:Alis-MacBook-Pro.local-62029-1427966310794-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59307-1427877990935-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:18:43,531 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:18:43,532 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:18:43,533 INFO - Router: Verifying the message...
 2015-04-02 16:18:43,533 ERROR - ID:Alis-MacBook-Pro.local-62029-1427966310794-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59307-1427877990935-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:18:43,534 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:18:43,534 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:18:43,535 INFO - Router: Verifying the message...
 2015-04-02 16:18:43,535 ERROR - ID:Alis-MacBook-Pro.local-62029-1427966310794-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59342-1427878022540-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQSessionExecutor.dispatch(ActiveMQSessionExecutor.java:134)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:205)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:27:18,888 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:27:18,888 INFO - MQServer: Listener is starting...
 2015-04-02 16:27:18,921 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:27:19,024 INFO - Router: Verifying the message...
 2015-04-02 16:27:19,026 ERROR - ID:Alis-MacBook-Pro.local-62626-1427966838547-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59342-1427878022540-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQSessionExecutor.dispatch(ActiveMQSessionExecutor.java:134)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:205)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:27:20,032 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:27:20,032 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:27:20,033 INFO - Router: Verifying the message...
 2015-04-02 16:27:20,033 ERROR - ID:Alis-MacBook-Pro.local-62626-1427966838547-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59342-1427878022540-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-02 16:27:21,040 INFO - MQServer: Listener: ON and Processing...
 2015-04-02 16:27:21,041 INFO - Router: Msg is to be verified.. 
 2015-04-02 16:27:21,041 INFO - Router: Verifying the message...
 2015-04-02 16:27:21,065 ERROR - ID:Alis-MacBook-Pro.local-62626-1427966838547-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59342-1427878022540-2:1:1:1:1
 java.lang.NullPointerException
	at id.bri.switching.app.Router.processISOMessage(Router.java:81)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:69)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
2015-04-08 14:19:45,914 INFO - MQServer: Listener is starting...
 2015-04-08 14:19:45,915 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:19:45,935 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:19:49,022 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:19:49,939 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:19:49,940 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:19:50,035 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:19:50,037 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:19:50,038 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:19:50,340 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:19:50,784 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:19:50,785 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:19:50,851 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:19:50,851 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:19:50,852 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:19:51,024 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:19:51,609 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:19:51,610 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:19:51,720 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:21:57,057 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:21:57,057 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:21:57,275 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:21:58,256 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:21:58,256 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:21:58,315 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:22:05,035 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:22:05,035 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:22:05,319 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:22:05,646 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:22:05,647 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:22:05,698 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:40:11,330 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:40:11,331 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:40:13,232 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:40:14,476 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:40:14,476 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:40:14,525 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:40:14,526 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:40:14,526 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:40:15,396 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:40:16,610 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:40:16,611 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:40:17,233 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:40:17,233 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:40:17,233 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:40:17,576 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:40:20,871 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:40:20,871 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:40:21,666 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:40:21,667 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:40:21,667 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:40:26,074 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:40:28,982 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:40:28,983 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:40:29,055 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:40:29,055 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:40:29,056 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:40:29,285 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:40:30,850 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:40:30,851 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:40:31,013 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:40:31,013 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:40:31,014 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:40:31,306 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:40:33,234 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:40:33,234 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:40:33,303 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:40:33,304 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:40:33,304 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:40:33,451 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:40:34,566 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:40:34,566 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:40:34,647 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:40:34,647 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:40:34,647 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:40:34,756 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:40:35,723 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:40:35,758 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:40:35,882 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:40:35,883 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:40:35,883 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:40:36,273 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:40:37,161 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:40:37,162 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:40:37,261 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:40:37,261 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:40:37,262 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:40:37,604 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:40:38,385 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:40:38,385 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:40:38,429 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:40:38,429 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:40:38,430 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:40:38,623 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:40:38,995 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:40:38,995 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:40:39,062 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:40:39,063 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:40:39,063 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:40:39,310 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:40:40,231 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:40:40,231 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:40:40,316 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:40:40,317 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:40:40,317 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:40:40,695 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:40:41,513 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:40:41,514 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:40:41,599 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:40:41,600 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:40:41,600 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:40:41,901 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:40:44,007 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:40:44,008 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:40:44,091 INFO - MQServer: Sending verification message: success. 
 2015-04-08 14:40:44,092 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 14:40:44,092 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 14:40:44,534 INFO - Router: Verifying the 0200 message...
 2015-04-08 14:40:45,447 INFO - Router:  PROC CODE = 303030.
 2015-04-08 14:40:45,448 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 14:40:45,633 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:31:59,217 INFO - MQServer: Connection to activeMQ is established...
 2015-04-08 16:31:59,842 INFO - MQServer: Listener is starting...
 2015-04-08 16:31:59,842 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:31:59,848 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:32:00,655 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:32:01,711 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:32:01,712 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:32:01,716 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:32:01,717 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:32:01,717 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:32:02,252 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:32:05,796 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:32:05,797 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:32:05,798 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:32:05,798 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:32:05,799 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:32:07,875 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:32:09,568 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:32:09,569 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:32:09,570 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:32:09,570 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:32:09,571 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:32:09,968 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:32:11,602 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:32:11,603 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:32:11,604 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:32:11,604 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:32:11,604 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:32:12,512 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:32:16,253 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:32:16,254 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:32:16,254 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:32:16,255 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:32:16,255 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:32:17,068 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:32:18,779 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:32:18,780 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:32:18,781 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:32:18,781 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:32:18,781 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:32:19,265 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:32:21,896 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:32:21,897 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:32:21,898 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:32:21,899 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:32:21,899 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:32:22,173 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:32:22,915 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:32:22,916 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:32:22,917 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:32:22,917 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:32:22,918 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:32:23,134 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:32:24,848 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:32:24,848 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:32:24,849 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:32:24,849 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:32:24,850 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:32:25,299 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:32:26,800 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:32:26,801 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:32:26,802 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:32:26,802 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:32:26,803 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:32:27,269 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:32:28,708 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:32:28,709 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:32:28,709 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:32:28,710 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:32:28,710 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:32:28,962 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:32:36,176 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:32:36,176 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:32:36,177 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:32:36,177 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:32:36,177 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:32:36,628 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:32:39,253 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:32:39,254 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:32:39,255 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:32:39,256 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:32:39,256 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:32:40,103 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:33:01,529 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:33:01,530 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:33:01,530 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:33:01,531 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:33:01,531 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:33:09,819 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:34:41,456 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:34:41,457 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:34:41,457 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:34:41,458 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:34:41,458 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:34:51,651 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:35:13,970 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:35:13,971 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:35:13,972 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:35:13,972 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:35:13,972 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:35:15,133 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:35:17,657 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:35:17,658 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:35:17,659 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:35:17,659 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:35:17,659 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:35:43,188 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:36:10,035 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:36:10,036 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:36:10,037 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:36:10,037 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:36:10,037 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:36:24,320 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:36:36,397 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:36:36,398 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:36:36,398 INFO - MQServer: Sending verification message: success. 
 2015-04-08 16:36:36,399 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 16:36:36,399 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 16:36:37,409 INFO - Router: Verifying the 0200 message...
 2015-04-08 16:36:43,494 INFO - Router:  PROC CODE = 303030.
 2015-04-08 16:36:43,495 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 16:36:43,496 INFO - MQServer: Sending verification message: success. 
 2015-04-08 17:00:39,201 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 17:00:39,224 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 17:00:39,373 INFO - Router: Verifying the 0200 message...
 2015-04-08 17:00:40,223 INFO - Router:  PROC CODE = 303030.
 2015-04-08 17:00:40,224 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 17:00:40,224 INFO - MQServer: Sending verification message: success. 
 2015-04-08 17:00:47,463 INFO - MQServer: Listener: ON and Processing...
 2015-04-08 17:00:47,463 INFO - Router: Msg is to be verified in router.. 
 2015-04-08 17:00:47,939 INFO - Router: Verifying the 0200 message...
 2015-04-08 17:00:48,396 INFO - Router:  PROC CODE = 303030.
 2015-04-08 17:00:48,397 INFO - Router: Response msg that's sent: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-08 17:00:48,397 INFO - MQServer: Sending verification message: success. 
 2015-04-13 11:22:26,568 INFO - MQServer: Connection to activeMQ is established...
 2015-04-13 11:22:26,631 INFO - MQServer: Listener is starting...
 2015-04-13 11:22:26,631 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:22:26,637 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:22:31,215 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:ubuntu-mocash-51619-1428312516109-2:1:1:1:15
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQSessionExecutor.dispatch(ActiveMQSessionExecutor.java:134)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:205)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:22:32,224 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:22:32,224 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:22:36,239 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:ubuntu-mocash-51619-1428312516109-2:1:1:1:15
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:22:37,248 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:22:37,248 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:22:41,266 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:ubuntu-mocash-51619-1428312516109-2:1:1:1:15
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:22:42,272 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:22:42,273 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:22:46,293 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:ubuntu-mocash-51619-1428312516109-2:1:1:1:15
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:22:47,299 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:22:47,299 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:22:51,317 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:ubuntu-mocash-51619-1428312516109-2:1:1:1:15
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:22:52,325 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:22:52,325 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:22:56,342 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:ubuntu-mocash-51619-1428312516109-2:1:1:1:15
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:22:57,349 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:22:57,349 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:23:01,369 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:ubuntu-mocash-51619-1428312516109-2:1:1:1:15
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:23:01,375 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:23:01,375 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:23:05,395 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-56428-1428893128248-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQSessionExecutor.dispatch(ActiveMQSessionExecutor.java:134)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:205)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:23:06,399 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:23:06,400 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:23:10,417 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-56428-1428893128248-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:23:11,424 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:23:11,424 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:23:15,443 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-56428-1428893128248-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:23:16,453 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:23:16,453 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:23:20,470 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-56428-1428893128248-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:23:21,479 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:23:21,479 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:23:25,493 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-56428-1428893128248-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:23:26,500 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:23:26,501 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:23:30,517 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-56428-1428893128248-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:23:31,523 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:23:31,523 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:23:35,541 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-56428-1428893128248-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:23:35,550 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:23:35,551 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:23:39,564 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-58634-1428894895645-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQSessionExecutor.dispatch(ActiveMQSessionExecutor.java:134)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:205)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:23:40,572 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:23:40,572 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:23:44,591 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-58634-1428894895645-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 25 more
2015-04-13 11:23:45,598 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:23:45,598 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:23:49,615 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-58634-1428894895645-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:23:50,622 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:23:50,623 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:23:54,642 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-58634-1428894895645-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:23:55,646 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:23:55,646 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:23:59,662 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-58634-1428894895645-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:24:00,670 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:24:00,670 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:24:04,690 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-58634-1428894895645-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:24:05,697 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:24:05,697 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:24:09,712 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-58634-1428894895645-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:24:09,717 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:24:09,717 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:24:13,743 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-58742-1428894990245-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQSessionExecutor.dispatch(ActiveMQSessionExecutor.java:134)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:205)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:24:14,750 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:24:14,750 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:24:18,766 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-58742-1428894990245-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:24:19,769 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:24:19,770 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:24:23,784 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-58742-1428894990245-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:24:24,792 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:24:24,792 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:24:28,810 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-58742-1428894990245-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:24:29,815 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:24:29,816 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:24:33,836 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-58742-1428894990245-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:24:34,847 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:24:34,848 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:24:38,860 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-58742-1428894990245-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:24:39,865 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:24:39,865 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:24:43,881 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-58742-1428894990245-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:24:43,888 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:24:43,888 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:24:47,904 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59156-1428895330827-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQSessionExecutor.dispatch(ActiveMQSessionExecutor.java:134)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:205)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:24:48,908 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:24:48,909 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:24:52,926 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59156-1428895330827-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:24:53,935 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:24:53,935 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:24:57,952 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59156-1428895330827-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:24:58,961 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:24:58,961 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:25:02,976 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59156-1428895330827-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:25:03,981 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:25:03,981 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:25:07,997 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59156-1428895330827-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:25:09,003 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:25:09,004 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:25:13,029 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59156-1428895330827-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:25:14,034 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:25:14,035 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:25:18,047 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-59156-1428895330827-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQMessageConsumer.iterate(ActiveMQMessageConsumer.java:1367)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:194)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:28:28,879 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:28:28,879 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:28:32,896 ERROR - ID:Alis-MacBook-Pro.local-63440-1428898946344-2:1:1:1 Exception while processing message: ID:Alis-MacBook-Pro.local-63916-1428899308666-2:1:1:1:1
 java.lang.RuntimeException: Cannot connect the database!
	at id.bri.switching.app.Logging.<init>(Logging.java:34)
	at id.bri.switching.app.Router.processISOMessage(Router.java:73)
	at id.bri.switching.mq.MQServer.onMessage(MQServer.java:125)
	at org.apache.activemq.ActiveMQMessageConsumer.dispatch(ActiveMQMessageConsumer.java:1233)
	at org.apache.activemq.ActiveMQSessionExecutor.dispatch(ActiveMQSessionExecutor.java:134)
	at org.apache.activemq.ActiveMQSessionExecutor.iterate(ActiveMQSessionExecutor.java:205)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:122)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:43)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)
	at java.lang.Thread.run(Thread.java:722)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1038)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:338)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2237)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor2.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:525)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:243)
	at id.bri.switching.app.Logging.<init>(Logging.java:26)
	... 10 more
Caused by: java.net.SocketException: Network is unreachable
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:391)
	at java.net.Socket.connect(Socket.java:579)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:213)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:297)
	... 24 more
2015-04-13 11:28:33,903 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:28:33,904 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:30:46,552 INFO - MQServer: Connection to activeMQ is established...
 2015-04-13 11:30:46,584 INFO - MQServer: Listener is starting...
 2015-04-13 11:30:46,584 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:30:46,587 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:30:47,053 INFO - Router: Verifying the 0200 message...
 2015-04-13 11:30:48,845 INFO - Router:  PROC CODE = 303030.
 2015-04-13 11:30:48,845 INFO - Router: Response from Router: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-13 11:30:48,849 INFO - MQServer: Sending verification message: success. 
 2015-04-13 11:30:48,850 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:30:48,851 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:30:48,915 INFO - Router: Verifying the 0200 message...
 2015-04-13 11:30:48,961 INFO - Router:  PROC CODE = 303030.
 2015-04-13 11:30:48,962 INFO - Router: Response from Router: 02107C14010022C1C00219   5188280232469701303030   126000000   126000000   115520000150324061801519   518828023246970100          223452          12346003180IDRIDR105                                                                                                      180
 2015-04-13 11:30:48,962 INFO - MQServer: Sending verification message: success. 
 2015-04-13 11:30:48,963 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:30:48,963 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:30:49,015 INFO - Router: Response from Router: 
 2015-04-13 11:30:49,015 INFO - MQServer: There is incoming message, but no response needed.
 2015-04-13 11:30:49,016 INFO - MQServer: Listener: ON and Processing...
 2015-04-13 11:30:49,016 INFO - Router: Msg is to be verified in router.. 
 2015-04-13 11:30:49,073 INFO - Router: Response from Router: 
 2015-04-13 11:30:49,074 INFO - MQServer: There is incoming message, but no response needed.
 